### ISSUE TITLE
[Crash / 1.19.2 / Forge] Paperdoll render issue with Snowy Spirit mod.

### ISSUE THREAD

**Issue by murao-kun** [tr7zw/PaperDoll#10]
This is what happening when I ride sledge with dog attached to them.
Mod link: https://www.curseforge.com/minecraft/mc-mods/snowy-spirit
![image](https://user-images.githubusercontent.com/110398598/185712609-8de74e6b-7bf2-47cb-a1a4-e82b19a9a97a.png)

**Comment by murao-kun** [comment]
Sane with create. 
![image](https://user-images.githubusercontent.com/110398598/185758776-d95c4313-27cc-41e3-b760-e27ae59bef8b.png)

**Comment by tr7zw** [comment]
Uhhhhm not 100% sure if I can fix this. Showing the entity that is being ridden turned out kinda complicated, and my guess with the first one is, that the lead has no clue how to render that properly. The second one I down know how it should be looking like, but having the entire train slapped in there is probably also not a good solution. I might have to add a blacklist to not render entire trains/Giants and other funky stuff.

**Comment by murao-kun** [comment]
Yeah, the option to print something like ids of things and having a black list should totally help that. But even better will be to replace it with just regular sitting animation.
